app/main.py

```diff
@@ -1,3 +1,56 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance | int") -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other.real)
+
+    def __iadd__(self, other: "Distance | int") -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other.real
+        return self
+
+    def __mul__(self, other: int) -> "Distance | None":
+        if isinstance(other, Distance):
+            return None
+        return Distance(self.km * other.real)
+
+    def __truediv__(self, other: int) -> "Distance | None":
+        if isinstance(other, Distance):
+            return None
+        return Distance(round(self.km / other.real, 2))
+
+    def __eq__(self, other: "Distance | int") -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        return self.km == other.real
+
+    def __lt__(self, other: "Distance | int") -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        return self.km < other.real
+
+    def __gt__(self, other: "Distance | int") -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        return self.km > other.real
+
+    def __le__(self, other: "Distance | int") -> bool:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        return self.km <= other.real
+
+    def __ge__(self, other: "Distance | int") -> bool:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        return self.km >= other.real
```

Comment:
u can use previous created methods here, and make this methods in one line

Reply:
Done